monitors/everyone.js

```diff
@@ -0,0 +1,41 @@
+const { Monitor } = require('klasa');
+const { Permissions: { FLAGS } } = require('discord.js');
+
+module.exports = class extends Monitor {
+
+	constructor(...args) {
+		super(...args, { ignoreOthers: false });
+	}
+
+	async run(msg) {
+		if (!msg.member ||
+			!msg.guild.configs.everyoneRole ||
+			!msg.mentions.roles.size ||
+			msg.guild.me.roles.highest.position <= msg.member.roles.highest.position ||
+			!msg.guild.me.permissions.has(FLAGS.MANAGE_ROLES) ||
+			!await msg.hasAtLeastPermissionLevel(1)
```

Comment:
Is there any particular reason not to use `botPerms` nor `permLevel`?

Reply:
Because it's a monitor.

Reply:
This could maybe be made cleaner by moving it to a shouldRun method, since it's determining whether it should run